tests/acceptance/pageObjects/accountPage.js

```diff
@@ -42,11 +42,19 @@ module.exports = {
     logout: function () {
       return this.waitForElementVisible('@logoutButton')
         .click('@logoutButton')
+    },
+    accountsPageVisible: async function () {
```

Comment:
Modify this function to give weather the page is visible rather than waiting for the page to be visible.